resources/views/Names.blade.php

```diff
@@ -4,11 +4,9 @@
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-
-    <title>Document</title>
-
-    <script src="https://cdn.tailwindcss.com"></script>
-
+    <!-- <script src="https://cdn.tailwindcss.com"></script> -->
+     @vite('resources/css/app.css') 
```

Comment:
das Laden von den Assets reicht wenn man das einmal im <head> Bereich des login.blade.php aufruft. das muss nicht überall importiert werden. Kann man hier also entfernen
https://laravel.com/docs/11.x/vite#loading-your-scripts-and-styles

Reply:
Ohne  @vite('resources/css/app.css')  funktioniert nicht